src/python/aqueduct_executor/operators/utils/utils.py

```diff
@@ -29,7 +32,7 @@ def read_artifacts(
     input_paths: List[str],
     input_metadata_paths: List[str],
     artifact_types: List[InputArtifactType],
-) -> List[InputArtifact]:
+) -> List[Any]:
```

Comment:
why use Any instead of expanding the union?

Reply:
It's because parameters can inputs to these operators, and parameters can have any type as long as it is jsonable. Based on https://stackoverflow.com/questions/51291722/define-a-jsonable-type-using-mypy-pep-526, it seems like we either call it an `object` or `Any`. I chose the latter to make this library simpler, the idea being that it's the responsibility of the caller to cast/check any results to the appropriate type, and the mypy checks you're working on should help catch any issues around this.